pkg/cache/v2/cluster_test.go

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2023 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cache_v2
+
+import (
+	"testing"
+
+	"github.com/agiledragon/gomonkey/v2"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	cluster_v2 "kmesh.net/kmesh/api/v2/cluster"
+	core_v2 "kmesh.net/kmesh/api/v2/core"
+	maps_v2 "kmesh.net/kmesh/pkg/cache/v2/maps"
+	"kmesh.net/kmesh/pkg/utils/hash"
+)
+
+func TestClusterFlush(t *testing.T) {
+	t.Run("cluster status is UPDATE", func(t *testing.T) {
+		patches1 := gomonkey.NewPatches()
+		patches2 := gomonkey.NewPatches()
+		patches1.ApplyFunc(maps_v2.ClusterUpdate, func(key string, value *cluster_v2.Cluster) error {
+			return nil
+		})
+		patches2.ApplyFunc(maps_v2.ClusterDelete, func(key string) error {
+			return nil
+		})
+		defer func() {
+			patches1.Reset()
+			patches2.Reset()
+		}()
+
+		cache := NewClusterCache()
+		cluster1 := &cluster_v2.Cluster{
+			ApiStatus:      core_v2.ApiStatus_UPDATE,
+			Name:           "ut-cluster1",
+			ConnectTimeout: uint32(30),
+			LbPolicy:       cluster_v2.Cluster_RANDOM,
+		}
+		cluster2 := &cluster_v2.Cluster{
+			ApiStatus:      core_v2.ApiStatus_UPDATE,
+			Name:           "ut-cluster2",
+			ConnectTimeout: uint32(60),
+			LbPolicy:       cluster_v2.Cluster_ROUND_ROBIN,
+		}
+		cache.SetApiCluster(cluster1.Name, cluster1)
+		cache.SetApiCluster(cluster2.Name, cluster2)
+		cache.Flush()
```

Comment:
I mean we should know that it really does write bpf map.

Reply:
I think it's up to the UT of `map_v2.ClusterUpdate`, to test if it actually writes the bpf map

Reply:
I mean according to the apiStatus, we should know whether this handler work as expected.  Am i clear?